### PR TITLE
Removed deprecated method ``get_center_point`` and parameters ``azimuth_label_scale``, ``number_scale_value``, ``label_scale``, ``scale_factor``, ``size``, ``x_min``, ``x_max``, ``delta_x``, ``y_min``, ``y_max``, ``delta_y``

### PR DESCRIPTION
**Pull request by kolibril13**
Depreciation purpose.
In VS Code, I searched for  "until 0.11" and deleted all references.

### CODE REVIEW

File: manim/mobject/vector_field.py [hunk 1]
```diff
@@ -536,14 +530,8 @@ def __init__(
...
     ):
-        self.x_range = x_range or [
-            kwargs.pop("x_min", None) or floor(-config["frame_width"] / 2),
-            kwargs.pop("x_max", None) or ceil(config["frame_width"] / 2),
-        ]
-        self.y_range = y_range or [
-            kwargs.pop("y_min", None) or floor(-config["frame_height"] / 2),
-            kwargs.pop("y_max", None) or ceil(config["frame_height"] / 2),
-        ]
+        self.x_range = x_range
```
Comment (kolibril13) [inline manim/mobject/vector_field.py:533]: @icedcoffeeee: Can you check if this is right?

File: manim/mobject/number_line.py [hunk 2]
```diff
@@ -163,12 +157,6 @@ def __init__(
         numbers_to_include: Optional[Iterable[float]] = None,
         **kwargs,
     ):
-        # deprecation
-        number_scale_value = kwargs.pop("number_scale_value", None)
-        if number_scale_value is not None:
-            self.font_size = number_scale_value * DEFAULT_FONT_SIZE * 0.75
-        else:
-            self.font_size = font_size
```
Comment (kolibril13) [inline manim/mobject/number_line.py:160]:
```suggestion
        self.font_size = font_size
```

File: manim/mobject/svg/brace.py [hunk 3]
```diff
@@ -165,11 +159,6 @@ def __init__(
         buff=0.2,
         **kwargs
     ):
-        label_scale = kwargs.pop("label_scale", None)
-        if label_scale:
-            self.font_size = label_scale * DEFAULT_FONT_SIZE
-        else:
-            self.font_size = font_size
```
Comment (kolibril13) [inline manim/mobject/svg/brace.py:162]:
```suggestion
        self.font_size = font_size
```

File: manim/mobject/coordinate_systems.py [hunk 4]
```diff
@@ -2607,14 +2590,6 @@ def __init__(
...
         **kwargs,
     ):
-        # deprecation
-        azimuth_label_scale = kwargs.pop("azimuth_label_scale", None)
-        if azimuth_label_scale:
-            self.azimuth_label_font_size = (
-                azimuth_label_scale * DEFAULT_FONT_SIZE * 0.75
-            )
-        else:
-            self.azimuth_label_font_size = azimuth_label_font_size
```
Comment (kolibril13) [inline manim/mobject/coordinate_systems.py:2593]:
```suggestion
        self.azimuth_label_font_size = azimuth_label_font_size
```

File: manim/mobject/svg/code_mobject.py [hunk 5]
```diff
@@ -182,12 +176,6 @@ def __init__(
             stroke_width=stroke_width,
             **kwargs,
         )
-        # deprecation handling
-        scale_factor = kwargs.pop("scale_factor", None)
-        if scale_factor:
-            self.font_size = DEFAULT_FONT_SIZE / 2 * scale_factor
-        else:
-            self.font_size = font_size
```
Comment (kolibril13) [inline manim/mobject/svg/code_mobject.py:179]:
```suggestion
            self.font_size = font_size
```

File: manim/mobject/svg/code_mobject.py [hunk 6]
```diff
@@ -182,13 +176,7 @@ def __init__(
             stroke_width=stroke_width,
             **kwargs,
         )
-        # deprecation handling
-        scale_factor = kwargs.pop("scale_factor", None)
-        if scale_factor:
-            self.font_size = DEFAULT_FONT_SIZE / 2 * scale_factor
-        else:
             self.font_size = font_size
```
Comment (kolibril13) [inline manim/mobject/svg/code_mobject.py:179]:
```suggestion
        self.font_size = font_size
```

File: manim/mobject/vector_field.py (hunk 1 above)
Comment (icedcoffeeee) [reply manim/mobject/vector_field.py:533]:
```suggestion
        self.x_range = x_range or [
            floor(-config["frame_width"] / 2),
            ceil(config["frame_width"] / 2),
        ]
```
I think it's supposed to look like this

File: manim/mobject/vector_field.py [hunk 7]
```diff
@@ -537,13 +531,10 @@ def __init__(
...
     ):
         self.x_range = x_range or [
-            kwargs.pop("x_min", None) or floor(-config["frame_width"] / 2),
-            kwargs.pop("x_max", None) or ceil(config["frame_width"] / 2),
-        ]
-        self.y_range = y_range or [
-            kwargs.pop("y_min", None) or floor(-config["frame_height"] / 2),
-            kwargs.pop("y_max", None) or ceil(config["frame_height"] / 2),
+            floor(-config["frame_width"] / 2),
+            ceil(config["frame_width"] / 2),
```
Comment (kolibril13) [inline manim/mobject/vector_field.py:535]: @icedcoffeeee : can you also make these changes for the other occurrences?

**Review by hydrobeam — CHANGES_REQUESTED** [review]
Suggestions concerning `font_size`

File: manim/mobject/vector_field.py [hunk 8]
```diff
@@ -537,13 +531,10 @@ def __init__(
...
         ]
+        self.y_range = y_range
```
Comment (icedcoffeeee) [inline manim/mobject/vector_field.py:537]:
```suggestion
        self.y_range = y_range or [
            floor(-config["frame_height"] / 2),
            ceil(config["frame_height"] / 2),
        ]
```

File: manim/mobject/vector_field.py [hunk 9]
```diff
@@ -727,14 +705,8 @@ def __init__(
...
     ):
-        self.x_range = x_range or [
-            kwargs.pop("x_min", None) or floor(-config["frame_width"] / 2),
-            kwargs.pop("x_max", None) or ceil(config["frame_width"] / 2),
-        ]
-        self.y_range = y_range or [
-            kwargs.pop("y_min", None) or floor(-config["frame_height"] / 2),
-            kwargs.pop("y_max", None) or ceil(config["frame_height"] / 2),
-        ]
+        self.x_range = x_range
```
Comment (icedcoffeeee) [inline manim/mobject/vector_field.py:708]:
```suggestion
        self.x_range = x_range or [
            floor(-config["frame_width"] / 2),
            ceil(config["frame_width"] / 2),
        ]
```

File: manim/mobject/vector_field.py [hunk 10]
```diff
@@ -727,14 +705,8 @@ def __init__(
...
+        self.y_range = y_range
```
Comment (icedcoffeeee) [inline manim/mobject/vector_field.py:709]:
```suggestion
        self.y_range = y_range or [
            floor(-config["frame_height"] / 2),
            ceil(config["frame_height"] / 2),
        ]
```

**Review by icedcoffeeee — COMMENTED** [review]
Some changes for default values in vectorfield.py

File: manim/mobject/vector_field.py (hunk 7 above)
Comment (kolibril13) [reply manim/mobject/vector_field.py:535]: Thanks a lot, @icedcoffeeee !

File: manim/mobject/number_line.py [hunk 11]
```diff
@@ -163,13 +157,7 @@ def __init__(
...
-
+        self.font_size = font_size
```
[the remark this answers is not in the document]
Comment (kolibril13) [reply manim/mobject/number_line.py:160]:
```suggestion
```

File: manim/mobject/svg/code_mobject.py [hunk 12]
```diff
@@ -182,13 +176,7 @@ def __init__(
...
             **kwargs,
         )
-        # deprecation handling
-        scale_factor = kwargs.pop("scale_factor", None)
-        if scale_factor:
-            self.font_size = DEFAULT_FONT_SIZE / 2 * scale_factor
-        else:
-            self.font_size = font_size
-
+        self.font_size = font_size
```
[the remark this answers is not in the document]
Comment (kolibril13) [reply manim/mobject/svg/code_mobject.py:179]: got it!

File: manim/mobject/number_line.py [hunk 13]
```diff
@@ -163,13 +157,6 @@ def __init__(
         numbers_to_include: Optional[Iterable[float]] = None,
         **kwargs,
     ):
-        # deprecation
-        number_scale_value = kwargs.pop("number_scale_value", None)
-        if number_scale_value is not None:
-            self.font_size = number_scale_value * DEFAULT_FONT_SIZE * 0.75
-        else:
-            self.font_size = font_size
```
Comment (hydrobeam) [inline manim/mobject/number_line.py:171]: I think you might've forgotten to re-add the font_size parameter here.

Comment (kolibril13) [reply manim/mobject/number_line.py:171]: here it is now. I hope the location is good.

**Comment by kolibril13** [comment]
@hydrobeam : Do you mind having a look why readthedocs fails?
Feel free to push the fix directly on this branch

**Review by behackl — APPROVED** [review]

**Pull request merged**